templates/BoaJavaHadoop.stg

```diff
@@ -29,7 +29,9 @@ public class <name> extends boa.runtime.BoaRunner {
 	}
 
 	public static void main(String[] args) throws Exception {
-		System.exit(org.apache.hadoop.util.ToolRunner.run(new <name>(), args));
+		<if(isLocal)>org.apache.hadoop.util.ToolRunner.run(new <name>(), args);
```

Comment:
Calling exit stops the JVM, while to display the results of the query, Evaluator needs the same JVM to keep on running. My intention is to make local mode slightly more responsive than cluster mode. I had two options
1. Run the BoaEvaluator, which puts the output at the given location (user has to give output location) and user can manually check the results

2. Run the BoaEvaluator, which puts the output at the given location (user may or may not give the output location, it is optional) and Evaluatory itself will display the results as part of running the query. Later on user may decide to check results manually. 

I opted for  option 2, but we can keep option 1 if required. I find 2 more convenient.